python/lsst/ts/mtaos/config_schema.py

```diff
@@ -240,6 +240,28 @@
       - type: "null"
     default: 900.0
 
+  enable_pointing_origin_compensation:
```

Comment:
I think "compensation" is not really the correct term here, please, consider renaming this "correction" or something on those line... Also, I would remove the "origin" as well, e.g. `enable_pointing_correction`.